3/series.py

```diff
@@ -0,0 +1,15 @@
+len = int(input())
+series = ""
+operation = True
+for i in range (1, len*2+1, 2):
+    series += f"{i}/{i+1}"
+    if i+1 == len*2:
+        pass
+    else:
+        if operation==True:
```

Comment:
Really? 

```suggestion
        if operation:
```
It doesn't make any sense to compare a boolean for True!!!